### PR TITLE
Fix spelling of the example abbreviation (e.g.)

### PR DESCRIPTION
**Pull request by waldyrious**
Most of the instances of this abbreviation already use the "e.g." form.
This commit changes the non-conforming ones to match.

Note that although this commit changes 29 files, almost all of these changes are in code comments. The only user-visible ones are in the README, and in localizable strings on the following six files:

- [src/vs/code/node/cliProcessMain.ts](https://github.com/microsoft/vscode/pull/74785/files#diff-c2fcc1d716c7f3f8e760e4923daba7ba)
- [src/vs/workbench/contrib/debug/common/debugSchemas.ts](https://github.com/microsoft/vscode/pull/74785/files#diff-2b34e7988e23ea73f5fed5bcb0bb2cfd)
- [src/vs/workbench/contrib/extensions/electron-browser/extensions.contribution.ts](https://github.com/microsoft/vscode/pull/74785/files#diff-9267db309226a9b7d772350586cce351)
- [src/vs/workbench/contrib/markers/browser/messages.ts](https://github.com/microsoft/vscode/pull/74785/files#diff-d99f77eba286403d5e3f2a2f6fc6f606)
- [src/vs/workbench/contrib/snippets/browser/snippets.contribution.ts](https://github.com/microsoft/vscode/pull/74785/files#diff-17ceef10bb7f25f818675b1c07f0aa01)
- [src/vs/workbench/services/keybinding/electron-browser/keybindingService.ts](https://github.com/microsoft/vscode/pull/74785/files#diff-15b94ff607048c9375b2982e7d6af63e)

### CODE REVIEW

**Comment by waldyrious** [comment]
By the way, I'm not making this change spuriously; My original goal was to propose a change one of these messages (the one in [src/vs/workbench/contrib/markers/browser/messages.ts](https://github.com/microsoft/vscode/pull/74785/files#diff-d99f77eba286403d5e3f2a2f6fc6f606)), to make it clearer and more explicit, like this:

```diff
-"Filter. Eg: text, **/*.ts, !**/node_modules/**");
+"Filter files. E.g.: text, **/*.ts, !**/node_modules/**");
```

but didn't want to change the "Eg." to "E.g." in one place while leaving the others in an inconsistent state.

**Comment by alexr00** [comment]
The files under src\typings are copied in, and shouldn't be changed since the changes will get overwritten.

File: src/vs/base/parts/tree/browser/tree.ts [hunk 1]
```diff
@@ -231,7 +231,7 @@ export interface IDataSource {
 	 *
 	 * You should not attempt to "move" an element to a different
 	 * parent by keeping its ID. The idea here is to have tree location
-	 * related IDs (eg. full file path, in the Explorer example).
+	 * related IDs (e.g.. full file path, in the Explorer example).
```
Comment (alexr00) [inline src/vs/base/parts/tree/browser/tree.ts]: Extra period.

**Review by alexr00 — CHANGES_REQUESTED** [review]

**Comment by waldyrious** [comment]
Thanks for the careful review, @alexr00. I'll fix and update the PR shortly.

**Review by alexr00 — APPROVED** [review]
Looks good to me!

**Comment by waldyrious** [comment]
@alexr00 thanks for the quick merge :) for future reference, was the extended message in this specific commit not particularly useful, or is there some guideline to not add additional details to commit messages?

**Comment by alexr00** [comment]
Concise details are always welcome!

**Pull request merged**